src/components/Explorer/DropdownContents.vue

```diff
@@ -27,6 +33,26 @@ export default defineComponent({
       type: Array as PropType<DropdownSortOrders[] | DropdownTargetedOptions[]>,
```

Comment:
```suggestion
      type: Array as PropType<String[]>,
```
にしてoptについてはMenusの中で対応関係を作ると汎用的なコンポーネントになると重ます